resources/assets/css/vedst.css

```diff
@@ -340,7 +340,10 @@ input[type=number],
     width: 100%;
     margin-left: -50%;
     z-index: 999999;
+}
 
+.alert-fixed{
+    position: fixed !important;
```

Comment:
important is bad code smell
last css rule overrides 
if you use this css class in combination with others just be sure, that this is the last one